util/bytes2sigchain.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/nknorg/nkn/pb"
+)
+
+// Base64ToHex convert base64 string input to hex string output
+func Base64ToHex(in []byte) (out []byte, err error) { // input format should be ':"balabala..."'
+	start, end := len(":\""), len(in)
```

Comment:
what about
start, end := len(":\""), len(in)-len("\"")
so that we can use end directly? Currently only end-1 is used.

Reply:
I pushed new commit again for better readability of Base64ToHex.